app/controllers/AnnotationIOController.scala

```diff
@@ -222,6 +237,9 @@ Expects:
   private def descriptionForNMLs(descriptions: Seq[Option[String]]) =
     if (descriptions.size == 1) descriptions.headOption.flatten.getOrElse("") else ""
 
+  private def wkUrlsForNMLs(wkUrls: Seq[Option[String]]) =
+    if (wkUrls.size == 1) wkUrls.headOption.flatten.getOrElse("") else ""
```

Comment:
I think this logic should be a bit different for this. If there are multiple uploaded files, and all have the same wkUrl, it should also be returned here. So maybe wkUrls.toSet.size ?